ReactTask/src/modules/Btns.modules.ts

```diff
@@ -0,0 +1,19 @@
+export interface IBtnCreateAppointment {
+  cliclChangeBox?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export interface IBtnPatients {
+  text: string;
+  active: boolean;
+}
+
+export interface IBtnPatientsHiddenMob {
+  text: string;
+  active: boolean;
+}
```

Comment:
Btns.model.ts it would be better to name file because it is model. 
But this is a lyrical digression. IBtnPatients & IBtnPatientsHiddenMob are the same, avoid dublication of code

Reply:
Okay, I changed name files and thanks, for noticing 2 identical interfaces